### ISSUE TITLE
[OTHER] Add TechStack to README.md

### ISSUE THREAD

**Issue by Snailedlt** [devicons/devicon#1422]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### What would you like to share?

Would be nice to have a `TechStack` or `Built With` section, where we list the technologies, languages, tools and frameworks that are used to develop, build and maintain the repo and website.
It could also be used as sort of a showcase of the icons by adding icons for each technology.

Maybe categorizing by what they're used for would be nice too?

Anyways, here are some of the technologies I know we use, but there are probably a few more that I'm missing:
- [Devicon.dev](https://devicon.dev) (`HTML`, `CSS`, `JavaScript`, `Angular`)
- To automate processes (`GitHub Actions`, `CI/CD`, `Python`, `Selenium`, `Gulp.js`)
- Icons (`SVG` and `JSON` editing)
- Documentation (`GitHub Wiki`, `Markdown`, `HTML`, `CSS`)

### Additional information

Please come with suggestions

**Comment by wvinson43** [comment]
do you mean add a section to the Readme like that or to the webpage?

**Comment by Snailedlt** [comment]
@wvinson43 Yes, sorry for not specifying. It should be added to the README.

Edited the issue description to specify it, thanks for asking :)

**Comment by wvinson43** [comment]
Ah cool, I may be able to help with that.  No promises, but I can give it a whack!

    On Friday, October 14, 2022 at 03:22:28 PM CDT, Jørgen Kalsnes Hagen ***@***.***> wrote:  
 
 


@wvinson43 Yes, sorry for not specifying. It should be added to the README.

Edited the issue description to specify it, thanks for asking :)

**Comment by wvinson43** [comment]
So to adjust the Read.me would I first initiate a PR?  I am pretty new to this!

**Comment by Snailedlt** [comment]
Great!
New to GitHub, or just this repo? I'll try and guide you either way,:)

So first create a fork, then create a new branch based on the `develop` branch, and edit the README file either locally or on GitHub. When you've edited it create a PR to the `develop` branch. 

If you did it correctly the only change in your PR should be in the README.md file. Good luck, and let me know if you need any more assistance :)

**Comment by wvinson43** [comment]
Not new to Github, but new to opensource.  I will give it a go later tonight!

**Comment by wvinson43** [comment]
I may have to postpone it a bit.  I will try to get to it ASAP!

**Comment by Panquesito7** [comment]
Any updates on this, @wvinson43? Do you need any help? Thanks. 🙂
